doc/tutorial/extending_theano.txt

```diff
@@ -259,7 +259,7 @@ Op Example
 
         def make_node(self, x):
             # check that the theano version has support for __props__
-            assert hasattr(self, '_props')
+            assert hasattr(self, '__props__')
```

Comment:
This was added when \__props__ was very new to make sure that people that tried the example with an older theano had an error.  Now it should be fine to remove it since \__props__ is old enough.

Otherwise, you should leave it as it was before.